testutils/store_mock.go

```diff
@@ -2,8 +2,12 @@ package testutils
 
 import (
 	"time"
+
+	"github.com/Azure/azure-container-networking/store"
 )
 
+var _ store.KeyValueStore = (*KeyValueStoreMock)(nil)
```

Comment:
curious: what is the purpose of this anonymous variable?

Reply:
this coerces `nil` in to type `*KeyValueStoreMock` and assigns it an anonymous interface type `store.KeyValueStore`. it has no run-time footprint, since it's a nil assigned to no variable, and it tells us at compile-time that the *Struct is an implementation of the Interface.